### ISSUE TITLE
Unable to verify the first certificate

### ISSUE THREAD

**Issue by rimiti** [TooTallNate/proxy-agents#99]
Hello,

In a previous version, it was possible to set `rejectUnauthorized: false` into the contructor object.
But in the new version (v5.0.0) it seems not possible, or not working.

I'm getting the below error:
```
unable to verify the first certificate
```

Is anyone know how "auto trust" certs ?
Thanks a lot 🙏

**Comment by gnyani** [comment]
Facing the same issue, rolled back to 3.0.0

**Comment by CedricBrandt** [comment]
Also affected

**Comment by rimiti** [comment]
@TooTallNate Could you please this issue? We are many to be blocked by this.

**Comment by rodrigoords** [comment]
+1

**Comment by krzysztof-cislo** [comment]
Is this project dead? No reply or update for one year already.

**Comment by gsongsong** [comment]
Might `process.env.NODE_TLS_REJECT_UNAUTHORIZED = '0'` help?

**Comment by TooTallNate** [comment]
This module has gone through a large refactor and modernization. I am closing this issue as a bit of house cleaning. If you feel that this issue still exists in the latest release, feel free to open a new issue.
